src/openrct2/ride/gentle/observation_tower.c

```diff
@@ -107,7 +107,6 @@ static void paint_observation_tower_base(paint_session * session, uint8 rideInde
 
     sint32 blockedSegments = 0;
     switch (trackSequence) {
-        case 0: blockedSegments = SEGMENTS_ALL; break;
```

Comment:
Seems to change the logic here. What if trackSequence is 0?

Reply:
It will return early in the if-block above. Case 0 is unreachable code.